repository-data/application/src/main/resources/hcm-config/configuration/frontend/cms-validators.yaml

```diff
@@ -18,3 +18,7 @@ definitions:
       jcr:primaryType: frontend:plugin
       plugin.class: uk.nhs.digital.ps.BlankStaticDropdownSelectionFieldValidator
       fieldDisplayName: Granularity
+
+    /hippo:configuration/hippo:frontend/cms/cms-validators/blank-related-link-field-forbidden:
```

Comment:
can you rename this one to "publicationsystem-blank-..."

And could you also rename "granularity" and "attachment" too?

Reply:
Shall I rename to:
"publicationsystem-blank-granularity"
"publicationsystem-blank-attachment"
"publicationsystem-blank-related-link"

or without that "...-blank-..." which will keep it consistent with the other two validators:
"publicationsystem-summary"
"publicationsystem-title"